### ISSUE TITLE
Sidebar stuck in FF 44.0

### ISSUE THREAD

**Issue by sgbotsford** [tabkit/tabkit2#100]
In Firefox 44 the TK2 side bar collapses to about 50 pixels (1/4" on a 23 inch display).  Repeated attempts to resize fail.  Manually setting minimum with to 250 px in settings fails.

Reverting to FF 43 remembers the 1/4" size, but it is now resizeable.

I attempted a re-install of TT2 as well as Tab highlighter.

**Comment by PikachuEXE** [comment]
I will wait for next ESR before trying to fix it
Since they might "fix" it in next version

**Comment by sgbotsford** [comment]
Not a problem.  I regressed to 43.  Post a note when it is fixed/fixes
itelf.

Respectfully,

Sherwood of Sherwood's Forests

Sherwood Botsford
Sherwood's Forests --  http://Sherwoods-Forests.com
780-848-2548
50042 Range Rd 31
Warburg, Alberta T0C 2T0

**Comment by patknight** [comment]
I agree with you waiting for ESR. Just adding a data point: FF44.02 on Linux and Windows 10 do not show this problem for me. FF44.01 and FF44.02 on Mac OS X with Retina display do show it for me.

**Comment by sgbotsford** [comment]
Non retina display -- Mac Pro 2012.

BTW:  Thanks so much for this kit.  I went once skipping some 10 versions
when FF broke tabkit.

Respectfully,

Sherwood of Sherwood's Forests

Sherwood Botsford
Sherwood's Forests --  http://Sherwoods-Forests.com
780-848-2548
50042 Range Rd 31
Warburg, Alberta T0C 2T0

**Comment by PikachuEXE** [comment]
Does this issue still exist with FF 45.x & latest version of Tab Kit?

**Comment by sgbotsford** [comment]
Yup.  Problem still there.

Any other information/tests/etc I can do to help you?

Respectfully,

Sherwood of Sherwood's Forests

**Comment by PikachuEXE** [comment]
OSX issue are hard to debug (I am on windows)
I do have a Mac but I rarely use it for debugging
I will try to look at this tomorrow

**Comment by PikachuEXE** [comment]
I still can't reproduce with my Mac (FF `45.0.1`)
Here are things you can do:

1. Try the new version which will be released soon `0.12.10`
2. Help me find out what happens when you have that issue (sidebar being not resizable):
  - Find out the console log to see if there is any error
  - Find out the tab bar's width 
    (maybe stuck at some value the code have set due a bug)

## Find out the console log to see if there is any error
- Install these extensions
  - https://addons.mozilla.org/en-US/firefox/addon/browser-console-button/
- Open new tab, enter "about:blank" in URL (to ensure no noise from loading browser content)
- Restart browser (FF)
- Open the console with that button
- Change filter to:
![1](https://cloud.githubusercontent.com/assets/1018543/14946902/b7c8ec34-105c-11e6-8eb1-5892358bb7d1.png)
![2](https://cloud.githubusercontent.com/assets/1018543/14946903/b8a514b6-105c-11e6-8f37-d734ad58d03a.png)
- Press clear
- Start resizing the tab bar (even it won't resize at the end)
- Upload the screenshot of the whole console window

## Find out the tab bar's width
- Install these extensions
  - https://addons.mozilla.org/en-US/firefox/addon/dom-inspector-6622/
  - https://addons.mozilla.org/en-US/firefox/addon/inspectthis/
- Restart after install (otherwise they won't work)
- Right click on a tab, click on "inspect"
  ![right](https://cloud.githubusercontent.com/assets/1018543/14946968/e4edfd48-105d-11e6-9079-cf3a2813ff8b.png)
- **Uncheck** "Show Anonymous Elements" (Otherwise it's showing too many stuff)
- Close the window
- (Again) Right click on a tab, click on "inspect"
- Find the entry with id "TabsToolbar"
  ![tabbar](https://cloud.githubusercontent.com/assets/1018543/14946991/3db15e5c-105e-11e6-8e1e-2f8442db7aca.png)
- Upload the screenshot of the whole inspector window

**Comment by sgbotsford** [comment]
Actually my test wasn't with FF45 but ff46.


I'll make a stab at this soon, but today I have 7 customer appointments and
10 orders to pack for the bus.  Not today

Respectfully,

Sherwood of Sherwood's Forests
